src/recipe.c

```diff
@@ -545,6 +546,12 @@ recipe_handler (gpointer user_data)
 
     switch (app_data->state) {
         case RECIPE_FETCH:
+            if (!app_data->stdin && BKR_ENV_EXISTS () && !BKR_RECIPE_IS_HEALTHY (app_data->recipe_url)) {
```

Comment:
Just a suggestion.  We're using the same code in 4 different places.  I'm wondering if it would be cleaner
and more maintainable to create an in-line function.
inline gboolean rstrnt_wait_for_beaker(AppData *app_data, gchar *err_msg);
if (rstrnt_wait_for_beaker(app_data, "* Recipe fetch: waiting on Beaker for 60 seconds\n")) {
    break;
}